alby.go

```diff
@@ -269,6 +269,67 @@ func (svc *AlbyOAuthService) LookupInvoice(ctx context.Context, senderPubkey str
 	return "", false, errors.New(errorPayload.Message)
 }
 
+func (svc *AlbyOAuthService) GetInfo(ctx context.Context, senderPubkey string) (alias, color, pubkey, network string, block_height uint32, block_hash string, err error) {
+	alias = "getalby.com"
+	app := App{}
+	err = svc.db.Preload("User").First(&app, &App{
+		NostrPubkey: senderPubkey,
+	}).Error
+	if err != nil {
+		svc.Logger.WithFields(logrus.Fields{
+			"senderPubkey": senderPubkey,
+		}).Errorf("App not found: %v", err)
+		return "", "", "", "", 0, "", err
+	}
+	tok, err := svc.FetchUserToken(ctx, app)
+	if err != nil {
+		return "", "", "", "", 0, "", err
+	}
+	client := svc.oauthConf.Client(ctx, tok)
+
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/getinfo", svc.cfg.AlbyAPIURL), nil)
+	if err != nil {
+		svc.Logger.WithError(err).Error("Error creating request /getinfo")
+		return "", "", "", "", 0, "", err
+	}
+
+	req.Header.Set("User-Agent", "NWC")
```

Comment:
can the request header be moved to the creation of the client? it'll always be constant, right?

Reply:
will be done in cleanup PR